Domains/4-Application/Connectors/CivilInfrastructureFramework/CifGeometricRules.ecschema.xml

```diff
@@ -1401,7 +1401,7 @@
             <Class class="EqualSpacePointRuleAspect"/>
         </Target>
     </ECRelationshipClass>
-    <ECEntityClass typeName="FilletRuleBaseAspect" modifier="Abstract">
+	<ECEntityClass typeName="FilletRuleCommonAspect">
```

Comment:
You can't delete (or rename) classes in a minor schema-change. You'll need to leave the class name the way it was while deprecating & hiding its properties. In parallel, you duplicate these properties into a new class with a different name.

Reply:
The original base class FilletRuleBaseAspect is not deleted. It is there from line no. 1429. with all the properties hidden and deprecated.